backend/internal/handlers/menu_items/menu_items.go

```diff
@@ -344,9 +352,20 @@ func (h *Handler) GetMenuItemReviews(c *fiber.Ctx) error {
 		userObjID = &parsedUserID
 	}
 
-	reviews, err := h.service.GetMenuItemReviews(objID, userObjID)
-	if err != nil {
-		return err
+	var reviews []review.ReviewDocument
+	var err error
+	sortParam := query.SortBy
+	// checks if the menu item reviews should be sorted
+	if ValidSortingParams[sortParam] {
+		reviews, err = h.service.GetMenuItemReviews(objID, userObjID, sortParam)
+		if err != nil {
+			return err
+		}
+	} else {
+		reviews, err = h.service.GetMenuItemReviews(objID, userObjID, "timestamp")
+		if err != nil {
+			return err
+		}
```

Comment:
Are we able to avoid code duplication here by changing how we pass the third parameter?
```
sortParam = "timestamp"
if (ValidSortingParams[query.SortBy]) {
    sortParam = query.SortBy
}

reviews, err = h.service.GetMenuItemReviews(objID, userObjID, sortParam)
```
as a minified example